lib/ansible/modules/cloud/azure/azure_rm_virtualnetwork_info.py

```diff
@@ -242,7 +241,9 @@ def exec_module(self, **kwargs):
             results = self.list_items()
 
         if is_old_facts:
-            self.results['ansible_facts']['azure_virtualnetworks'] = self.serialize(results)
+            self.results['ansible_facts'] = {
+                'azure_virtualnetworks': self.serialize(results)
+            }
         self.results['virtualnetworks'] = self.curated(results)
```

Comment:
Only on _info calls.

Reply:
This has also been in `stable-2.8`: https://github.com/ansible/ansible/blob/stable-2.8/lib/ansible/modules/cloud/azure/azure_rm_virtualnetwork_facts.py#L209